DEVELOPMENT.md

```diff
@@ -356,6 +356,25 @@ changes to existing stuff are required for a new feature.
 Prior to pgai v0.4.0, Python dependencies were installed system-wide. Until pgai versions 0.1 - 0.3 are deprecated
 [old dependencies](./src/old_requirements.txt) are installed system-wide.
 
+#### Release the extension
+
+1. Run `just ext clean-sql` in order to remove any prior generated dev sql.
+2. Edit the [build.py](projects/extension/build.py).
+   - Set the version to be released as the first element of the array returned by `versions` function. I.e. `"0.7.0",  # released`.
+3. Run `just ext build-release`. This will generate, among other tasks, the final .sql files in `projects/extension/sql` and update the version in [__init__.py](projects/extension/ai/__init__.py).
+4. As those files are git-ignored, you need to add those to the git index by forcing the addition. 
+   - Run `git add -f projects/extension/sql/ai--*<YOUR_NEW_VERSION_HERE>.sql`.
+5. Craft the release notes for the new version of the extension. Those should be added into [projects/extension/RELEASE_NOTES.md](projects/extension/RELEASE_NOTES.md).
+   - As we are not using release-please, those notes are not automatically generated. Instead, we need to manually add those.  
+     You can use the output of the following command to craft the release notes:
+     ```shell
+     git log $(git describe --tags --abbrev=0 --match "extension-*")..origin/main --pretty=format:"%s ([%h](https://github.com/timescale/pgai/commit/%h))" projects/extension
+     ```
+6. Create a PR with all the modified files. Use a commit message such as `chore: release extension <YOUR_NEW_VERSION_HERE>`.
```

Comment:
maybe we could add the options (chore, feat, docs?

Reply:
I don't get exactly what you mean here. I added `chore` as the exact prefix you should use. At least, this is how we have been doing in the past with all releases.